docs/ovms_quickstart.md

```diff
@@ -60,11 +60,10 @@ docker run -d -u $(id -u):$(id -g) -v $(pwd)/model:/models/face-detection -p 900
 --model_path /models/face-detection --model_name face-detection --port 9000 --plugin_config '{"CPU_THROUGHPUT_STREAMS": "1"}' --shape auto
 ```
 
-OpenVINO Model Server expects a particular folder structure  for models -  in this case it will be mounted as: 
-`/models/face-detection/1`
+OpenVINO Model Server expects a particular folder structure for models - in this case `model` directory should have following content: 
 
 ```bash
-models/
+model/
```

Comment:
Why? 😛 
Don't we use name `models` as a top-level directory in other examples?

Reply:
model for local directory and models in OVMS.
In this example output for curl command is: -o model/1/face-detection-retail-0004.bin

Change it?

Reply:
but here you have: `model/face-detection/1/face-detection-retail-0004.bin`
shouldn't it be either:
 `models/face-detection/1/face-detection-retail-0004.bin` or
 `model/1/face-detection-retail-0004.bin`

Reply:
You right. I had a bug in code so it didn't catch it. Good eye! 
Fixed